specs/GetFavicon.md

```diff
@@ -0,0 +1,79 @@
+# Background
+A Favicon is an asset which is a part of a  webpage, and typically displayed on each tab. Developers would
+like to have an API which allows them to retrieve the Favicon of a webpage, if it has one, as well as get an update whenever
+the favicon has changed.
+
+# Description
+We propose a new Webview2 event which would allow developers access the current Favicon of a page, 
+as well as be notified when the favicon changes.
+
+# Examples
+## Win32 C++ Registering a listener for favicon changes
+    ```cpp
+        CHECK_FAILURE(m_webView->add_FaviconUriChanged(
+            Microsoft::WRL::Callback<ICoreWebView2FaviconUriChangedEventHandler>(
+            [this](ICoreWebView2* sender, IUnknown* args) -> HRESULT
+            {
+                
+                LPWSTR value;
```

Comment:
This should be `wil::unique_cotaskmem_string value;` so we don't leak the string